WEB_SERVICE/WEB/_MEDIUM/HSTS_HEADER/description.md

```diff
@@ -0,0 +1,12 @@
+This vulnerability indicates that the server does not support the HTTP Strict Transport Security (HSTS) header, or it is misconfigured.
+
+HSTS helps protect against protocol downgrade attacks and cookie hijacking by ensuring that browsers always connect using HTTPS.
+
+If HSTS is not configured properly, attackers could potentially force the use of an insecure HTTP connection.
+
+**Example Scenario:**
+An attacker could intercept a request to the server, forcing it to use HTTP instead of HTTPS, exposing sensitive data.
+
+**Mitigation:**
+- Ensure that the HSTS header is properly configured with a sufficient max-age.
+- Apply the header across all subdomains if applicable.
```

Comment:
What is the difference between this KB and the one in hardening , like why we will need two ?

Reply:
The focus of this one is Vulnerability Context, while the one in hardening is focused on Technical Explanation. Should we only keep one, I don't know to be honest.